wifiphisher/pywifiphisher.py

```diff
@@ -605,6 +629,12 @@ def start(self):
             time.sleep(1)
             self.stop()
 
+        if not args.no_mac_randomization:
+            print ("[{0}!{1}]" + ap_iface.get_name() + ' mac address becomes '
+                     + ap_iface.get_current_mac()).format(R, W)
+            print ("[{0}!{1}]" + mon_iface.get_name() + ' mac address becomes '
+                     + mon_iface.get_current_mac()).format(R, W)
```

Comment:
The problem is that in case of `--nojamming`, the user will see "wlan0 mac address becomes 00:00:00:ad:cb:f2" twice.